filebeat/module/nats/log/_meta/fields.yml

```diff
@@ -0,0 +1,5 @@
+- name: log
+  type: group
+  description: >
+    Nats log files
+  release: beta
```

Comment:
Can you add here one line with `fields:`. This should fix the tests. That `fields:` is required is more an issue of our generation script and we should clean that up. We just didn't have a case yet that a module does have no additional fields, but this is great.

Reply:
Thanks's for taking the time to catch it! Will update it!